zk/l1infotree/l2_info_tree_syncer.go

```diff
@@ -0,0 +1,122 @@
+package l1infotree
+
+import (
+	"context"
+	"encoding/json"
+	"github.com/ledgerwatch/erigon/eth/ethconfig"
+	zkTypes "github.com/ledgerwatch/erigon/zk/types"
+	jsonClient "github.com/ledgerwatch/erigon/zkevm/jsonrpc/client"
+	"github.com/ledgerwatch/log/v3"
+	"sync/atomic"
+	"time"
+)
+
+const (
+	EXIT_ROOT_TABLE = "zkevm_getExitRootTable"
+)
+
+type L2InfoTreeSyncer struct {
```

Comment:
I think this name is a bit misleading. Can we change the name? Smth like `L2RpcL1InfoTreeSyncer`.

Reply:
Or L1InfoTreeRpcSyncer